libcontainer/cgroups/fs/pids.go

```diff
@@ -20,20 +20,14 @@ func (s *PidsGroup) Apply(path string, _ *configs.Resources, pid int) error {
 }
 
 func (s *PidsGroup) Set(path string, r *configs.Resources) error {
-	if r.PidsLimit != 0 {
```

Comment:
Please change `PidsLimit` to a `*int64` pointer, then check `if r.PidsLimit != nil`.

Reply:
I've attempted this in a separate commit